animals_and_tips.html

```diff
@@ -126,115 +126,115 @@ <h2>Hover on each image to discover information about them</h2>
 <div class="container">
 	<img src="images/placeholder.png" alt="img" class="image">	
 	<div class="overlay overlayLeft">
-		<div class="text"><h1>Lion</h1><h2>Panthera Leo</h2><p>Stand your ground, retreating very slowly, continuing to face the lion while clapping your hands, shouting and waving your arms around to make yourself look bigger. Most charges are mock charges, so you will usually be fine. And hold your ground! Never run or turn your back.
+		<div class="text"><h1>Lion</h1><h2>Panthera Leo</h2><p>If in contact with a lion, stand your ground, retreating very slowly, continuing to face the lion while clapping your hands, shouting and waving your arms around to make yourself look bigger. Most charges are mock charges, so you will usually be fine. And hold your ground! Never run or turn your back.

```

Comment:
Information and spelling mistakes were correct for some of the information about the animals everything else was correct